python/test/test_install_gdal.py

```diff
@@ -0,0 +1,25 @@
+import random
+
+from pyspark.sql.functions import abs, col, first, lit, sqrt
+
+from .context import api
+from .utils import MosaicTestCaseWithGDAL
+
+
+class TestInstallGDAL(MosaicTestCaseWithGDAL):
```

Comment:
I need to remove this file.
tests with GDAL are super annoying since they require the installation script to have run.
Since the tests in python all they do is verify expressions we properly defined syntactically and nothing else all functionality is tested in scala.
For now I will leave GDAL untested in python. We need a better mechanism for GDAL. 
Probably through action steps.